bounties/pip/parlai/1/vulnerability.json

```diff
@@ -0,0 +1,50 @@
+{
+    "PackageVulnerabilityID": "1",
+    "DisclosureDate": "2020-12-27",
+    "AffectedVersionRange": "*",
+    "Summary": "Arbitrary Code Execution",
+    "Contributor": {
+      "Discloser": "Anon-Artist",
+      "Fixer": ""
+    },
+    "Package": {
+      "Registry": "pip",
+      "Name": "ParlAI",
```

Comment:
The test is failing because the name of the `directory` doesn't match the value of the `package`. Please rename the directory to `ParlAI` to match the value. :+1: